feature/bgp/policybase/otg_tests/import_export_multi_test/import_export_multi_test.go

```diff
@@ -286,12 +299,15 @@ func configureImportExportMultifacetMatchActionsBGPPolicy(t *testing.T, dut *ond
 		}
 	}
 	communitySetNestedReject.SetCommunityMember(cs2)
-	communitySetNestedReject.SetMatchSetOptions(matchInvert)
+	if deviations.BGPConditionsMatchCommunitySetUnsupported(dut) {
```

Comment:
See above, this doesn't appear to do what the intent of the deviation is as written.

Reply:
Thanks for the reference, now I get it!